x-pack/test/accessibility/apps/ml_embeddables_in_dashboard.ts

```diff
@@ -0,0 +1,133 @@
+/*
```

Comment:
The comment from the functional dashboard embeddables test files apply here in the a11y version as well, since it's basically a copy.

Reply:
Updated here [`673eafe` (#94396)](https://github.com/elastic/kibana/pull/94396/commits/673eafe202519b5851085c4ae0cfac9a74d6d604)